ee/clickhouse/models/property.py

```diff
@@ -161,18 +162,12 @@ def parse_prop_clauses(
                     f"{property_operator} 0 = 13"
                 )  # If cohort doesn't exist, nothing can match, unless an OR operator is used
             else:
-
-                if person_properties_mode == PersonPropertiesMode.EXCLUDE:
-                    person_id_query, cohort_filter_params = format_cohort_subquery(
-                        cohort, idx, custom_match_field=f"{person_id_joined_alias}"
-                    )
-                    params = {**params, **cohort_filter_params}
-                    final.append(f"{property_operator} {person_id_query}")
-                else:
-                    person_id_query, cohort_filter_params = format_filter_query(cohort, idx)
-                    params = {**params, **cohort_filter_params}
-                    final.append(f"{property_operator} {table_name}distinct_id IN ({person_id_query})")
-        elif prop.type == "person" and person_properties_mode != PersonPropertiesMode.EXCLUDE:
+                person_id_query, cohort_filter_params = format_cohort_subquery(
+                    cohort, idx, custom_match_field=f"{person_id_joined_alias}"
+                )
+                params = {**params, **cohort_filter_params}
+                final.append(f"{property_operator} {person_id_query}")
+        elif prop.type == "person" and person_properties_mode != PersonPropertiesMode.USING_DIRECT_QUERY:
```

Comment:
I removed the if condition altogether because it is confusing. Caused 2 test failures